charts/finrem-ns/values.yaml

```diff
@@ -1,15 +1,10 @@
 java:
+  aadIdentityName: finrem
+  ingressHost: "finrem-ns-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
   applicationPort: '8086'
   livenessPath: '/health'
-  environment:
-      AUTH_PROVIDER_SERVICE_CLIENT_MICROSERVICE: 'finrem-notification-service'
-      REFORM_SERVICE_NAME: 'ns'
-      AUTH_PROVIDER_SERVICE_CLIENT_TOKENTIMETOLIVEINSECONDS: '900'
-      REFORM_TEAM: 'finrem'
-  image: 'https://hmcts.azurecr.io/hmcts/finrem-notification-service:latest'
   keyVaults:
     finrem:
       resourceGroup: finrem
```

Comment:
this field is no longer required after upgrading the chart version

Reply:
`resourceGroup` removed